remote-query/pom.xml

```diff
@@ -39,7 +39,7 @@
 
         <!-- Defines the main class to be execute when running java -jar command -->
         <main.class.remote-query>org.jboss.as.quickstarts.datagrid.hotrod.query.AddressBookManager</main.class.remote-query>
-        <version.org.infinispan>7.0.0.Final</version.org.infinispan>
+        <version.org.infinispan>7.1.0-SNAPSHOT</version.org.infinispan>
```

Comment:
Oops, then it will have to be the infinispan version we release this week. Let's hold this PR until then.